src/common/components/AddTodoBtn/AddTodoBtn.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from "react";
 import { Button } from "@mui/material";
 import { UI_TEXT } from "../Components.Constants";
+import AddTodoModal from "../AddTodoModal/AddTodoModal";
 
 const AddTodoBtn: React.FC = () => {
   const [open, setOpen] = useState<boolean>(false);
```

Comment:
there is a naming convention for boolean values - beginning from **is** (rarely **are**)

isOpen, isCreated etc